build/shade/_k-standard-goals.shade

```diff
@@ -40,6 +40,15 @@ default SAMPLES_PROJECT_GLOB = "samples/*/project.json"
   }
 }
 
+#--for-ci
+  @{
+    Configuration = "Release";
+    E("Configuration", Configuration);
+    E("KOREBUILD_ADD_ASSEMBLY_INFO", "1");
+    E("KOREBUILD_DOTNET_PACK_OPTIONS", " --serviceable");
```

Comment:
Should this be additive? What if CI has additional options set in KOREBUILD_DOTNET_PACK_OPTIONS?